content/operate/kubernetes/deployment/helm.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: Install Redis Enterprise Helm chart
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: Install the Redis Enterprise for Kubernetes version 7.8.Wisconsin using helm charts.
+linkTitle: Helm
+weight: 11
+---
+
+Helm charts provide a simple way to install the Redis Enterprise for Kubernetes operator in just a few steps. For more information about Helm, go to [https://helm.sh/docs/](https://helm.sh/docs/).
+
+{{<note>}} This feature is currently in public preview and is not supported on production workloads. Only new installations of the Redis operator are supported at this time. The steps for [creating the RedisEnterpriseCluster (REC)]({{<relref "operate/kubernetes/deployment/quick-start#create-a-redis-enterprise-cluster-rec">}}) and other custom resources remain the same.{{</note>}}
+
+## Prerequisites
+
+- A [supported distribution]({{< relref "/operate/kubernetes/reference/supported_k8s_distributions.md" >}}) of Kubernetes.
+- At least three worker nodes.
+- [Kubernetes client (kubectl)](https://kubernetes.io/docs/tasks/tools/).
+- [Helm 3.10 or later](https://helm.sh/docs/intro/install/).
+
+### Example values
+
+The steps below use the following placeholders to indicate values you must provide: 
```

Comment:
Thinking, maybe this sentence better avoid using the term "values" (in "values you must provide"), to not create confusion with helm values. Can use "parameters" or even "command line parameters".